core/jobs/decorators/validation_decorators.py

```diff
@@ -62,7 +62,7 @@
     Iterator[
         Tuple[
             model_property.PropertyType,
-            List[Type[base_models.BaseModel]]
+            Sequence[Type[base_models.BaseModel]]
```

Comment:
Changed it from List to sequence, because in other places we are providing a sub-type of this type and List cannot allow subtyping due to its invariant property!. So, to allow subtyping  i used Sequence.